package.json

```diff
@@ -11,7 +11,7 @@
     "build": "gatsby build",
     "serve": "gatsby serve",
     "serve:fresh": "yarn clean; yarn build; yarn serve",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md,mdx}\"",
```

Comment:
We didn't check mdx files before :grimacing: hence the dense formatting changes in this PR